packages/react/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -226,6 +227,8 @@ export interface MultiSelectProps<ItemType>
    */
   items: ItemType[];
 
+  itemsWithSelectAll: ItemType[];
```

Comment:
Add to MultiSelect.propTypes and also would good to insert a description, so when the user reads the Overview page it has more context.